### PR TITLE
Remove `Request` and `Response` classes

### PR DESCRIPTION
**Pull request by andre-merzky**
It turned out that the `Request` and `Response` classes create some significant overhead on high frequency channels.  Since the classes are really only a thin convenience wrapper around some simple data structure (`dict`), this PR opts for removing them.  

The PR also cleans up some logging calls in the ZMQ code and also cleans up the `RADICAL_BASE` interpretation.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 name     = 'radical.utils'
 mod_root = 'src/radical/utils/'
 
+path = None
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply setup.py:25]: Pip changed the way it installs module from source trees and it's badly messing with our setup.  That will need fixing - but you are right, that is not for this PR, will remove.

[the remark this answers is not in the document]
Comment (andre-merzky) [reply setup.py]: ack, reverted. Thanks for catching!

File: src/radical/utils/zmq/bridge.py [hunk 2]
```diff
@@ -39,9 +40,13 @@ def __init__(self, cfg):
         self._uid     = self._cfg.uid
         self._pwd     = self._cfg.path
         self._log     = Logger(name=self._uid, ns='radical.utils',
-                               level=self._cfg.log_lvl, path=self._pwd)
+                               level='DEBUG', path=self._pwd)
+                             # level=self._cfg.log_lvl, path=self._pwd)
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/utils/zmq/bridge.py]: Thanks again, reverted.

**Comment by andre-merzky** [comment]
Thanks for the review.  I addressed those points and would appreciate another feedback round. :-)

**Review by mtitov — COMMENTED** [review]
All looks good, just have one comment regarding `DefaultConfig`

File: src/radical/utils/config.py [hunk 3]
```diff
@@ -411,6 +411,17 @@ class DefaultConfig(Config, metaclass=Singleton):
...
+                   'log_lvl'    : 'error',
+                   'log_tgt'    : 'stderr',
+                   'log_dir'    : '.',
+                   'report'     : True,
+                   'report_tgt' : 'stderr',
+                   'report_dir' : '.',
+                   'profile'    : True,
+                   'profile_dir': '.',
+                 }
+
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/utils/config.py:424]: ack, I removed those lines.  Thanks.

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**